backend/eln/eln-core/src/test/resources/com.epam.indigoeln.eln.entity/template_entity.json

```diff
@@ -0,0 +1,67 @@
+{
```

Comment:
Do we need this file in Git? If it's just to see the output, please generate file somewhere inside "build" directory so it's not included